unicef/iogt
# Community moderators can now add Canned Responses

Closes #37 

### Testing instructions
- Add canned responses from the admin site
- Login as moderator
- See the Canned Responses dropdown in the comment form
